debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/MMObjectAccessBarrier_V1.java

```diff
@@ -49,9 +50,18 @@ protected MMObjectAccessBarrier_V1() throws CorruptDataException
 		if(J9BuildFlags.gc_compressedPointers) {
 			try {
 				J9JavaVMPointer vm = J9RASHelper.getVM(DataType.getJ9RASPointer());
-				shift = vm.compressedPointersShift().intValue();
+
+				/* use reflection to access compressedPointersShift which will not exist if build is default */
+				Class J9JavaVMPointerClass = Class.forName("com.ibm.j9ddr.vm29.pointer.generated.J9JavaVMPointer"); //$NON-NLS-1$
+				UDATA shiftUdata = (UDATA)J9JavaVMPointerClass.getMethod("compressedPointersShift").invoke(vm); //$NON-NLS-1$
+				shift = shiftUdata.intValue();
 			} catch (CorruptDataException cde) {
 				raiseCorruptDataEvent("Error initializing the object access barrier", cde, true);
+			} catch (Exception e) {
+				/* internal error caused by reflection */
+				InternalError err = new InternalError(e.toString());
+				err.initCause(e);
+				throw err;
```

Comment:
Can this throw a CorruptDataException instead of InternalError?  If we can't find the `compressedPointersShift`, there is an issue with the data.

The rest of the DDR code will likely handle CDE better than IE.